user/firejail/files/tmpfiles.conf

```diff
@@ -0,0 +1,3 @@
+# Populate /etc/firejail with default application security profiles
+
+C /etc/firejail - - - - /usr/share/firejail
```

Comment:
this is probably bad because it will never get update once populated once

Reply:
Yes you're right. I looked further into this and it is even worse. Built as I did, Firejail will never look at these files in /etc/. There is no need for /etc/firejail because profiles can be overridden in ~/.config/firejail/.   I will remove this. It makes no sense.